### PR TITLE
Moving to use establishment name rather establishment ID for home page

### PR DESCRIPTION
**Pull request by andrewrlee**
### Context

No ticket - addressing some tech debt to help aid  the move to single URL.

### Intent

Moving the home page over to use establishment name rather than establishment ID

### Considerations

n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/content/establishmentData.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
   "792": {
     "name": "berwyn",
     "displayName": "HMP Berwyn",
-    "homePageLinksTitle": "Popular topics",
```
Comment (andrewrlee) [inline server/content/establishmentData.json:5]: removed this configuration as they don't vary

**Review by jonhindle — APPROVED** [review]
lgtm

**Review by davidthomas22 — COMMENTED** [review]
LGTM

**Review by davidthomas22 — COMMENTED** [review]
LGTM

**Pull request merged**